superset-frontend/src/dashboard/components/gridComponents/Markdown.jsx

```diff
@@ -65,14 +69,16 @@ const propTypes = {
 
 const defaultProps = {};
 
-const markdownPlaceHolder = `# ✨Markdown
+const MARKDOWN_PLACE_HOLDER = `# ✨Markdown
 ## ✨Markdown
 ### ✨Markdown
 
 <br />
 
 Click here to edit [markdown](https://bit.ly/1dQOfRK)`;
 
+const MARKDOWN_ERROR_MESSAGE = t('This markdown component has an error.');
```

Comment:
i name rename 2 pre-defined markdown constant as `MARKDOWN_ERROR_MESSAGE` and `MARKDOWN_PLACE_HOLDER `